### ISSUE TITLE
Error: Uncaught (in promise): null (reopen Issue #60)

### ISSUE THREAD

**Issue by moritzvieli** [NativeScript/nativescript-camera#107]
### Which platform(s) does your issue occur on?
- iOS/Android/Both 
- iOS/Android versions 
- emulator or device. What type of device?
--> HTC real device with Android 6.0.1

### Please, provide the following version numbers that your issue occurs with:
tns --version
3.4.3

tns-core-modules
3.4.1

nativescript-camera
4.0.2

### Please, tell us how to recreate the issue in as much detail as possible. 
Describe the steps to reproduce it.

import * as camera from 'nativescript-camera';

camera.takePicture({
                    width: 600,
                    keepAspectRatio: true,
                    // By activating, camera takes on HTC don't work anymore (real device)
                    //saveToGallery: true
                })
                .then((picture) => {
                   // It's working until now
                imageSourceModule.fromAsset(response).then((value) => {
                    this.image = value.toBase64String('jpeg');
                    this.imageWidth = value.width;
                    this.imageHeight = value.height;
                });

--> Error: Error: Uncaught (in promise): null

Basically it's the same problem as described in Issue #60. This code worked with plugin version 3.0.0 but not anymore with 4.0.2.

**Comment by moritzvieli** [comment]
-> Can be closed in favor of reopened issue #60
